### PR TITLE
Clip_vit_large14 and t5 models

### PR DESCRIPTION
**Pull request by dan-garvey**

### CODE REVIEW

**Comment by dan-garvey** [comment]
@IanNod I opted to just test a smaller t5 model, as there isn't anything particularly unique to the xxl

File: models/turbine_models/tests/sd_test.py [hunk 1]
```diff
@@ -67,6 +67,38 @@
 
 
 class StableDiffusionTest(unittest.TestCase):
```
Comment (IanNod) [inline models/turbine_models/tests/sd_test.py:69]: Only testing that we can compile. Can you also check runtime outputs match torch to some tolerance?

File: models/turbine_models/custom_models/sd_inference/clip.py [hunk 2]
```diff
@@ -6,6 +6,7 @@
 
 import os
 import sys
+import re
```
Comment (IanNod) [inline models/turbine_models/custom_models/sd_inference/clip.py:9]: nit: looks unused?

Comment (dan-garvey) [reply models/turbine_models/custom_models/sd_inference/clip.py:9]: line 166

Comment (dan-garvey) [reply models/turbine_models/custom_models/sd_inference/clip.py:9]: (before my patch this file errored on undefined re)

File: models/turbine_models/custom_models/sd_inference/clip_runner.py [hunk 3]
```diff
@@ -52,49 +53,125 @@ def run_clip(
...
-    )
-    text_input = tokenizer(
-        prompt,
-        padding="max_length",
-        max_length=tokenizer.model_max_length,
-        truncation=True,
-        return_tensors="pt",
-    )
+            url = "http://images.cocodataset.org/val2017/000000039769.jpg"
+            image = Image.open(requests.get(url, stream=True).raw)
```
Comment (IanNod) [inline models/turbine_models/custom_models/sd_inference/clip_runner.py:128]: Why do we need an image?

**Review by IanNod — APPROVED** [review]

File: models/turbine_models/custom_models/sd_inference/clip_runner.py (hunk 3 above)
Comment (dan-garvey) [reply models/turbine_models/custom_models/sd_inference/clip_runner.py:128]: it takes image as input

Comment (dan-garvey) [reply models/turbine_models/custom_models/sd_inference/clip_runner.py:128]: may be optional, I have to rework this in another patch anyway

**Pull request merged**